src/core/adaptersController.ts

```diff
@@ -13,6 +13,9 @@ export class AdaptersController {
   private protocolTokens:
     | Promise<Map<Chain, Map<string, IProtocolAdapter>>>
     | undefined
+  private rewardTokens:
```

Comment:
I thought this wasn't needed with the decorator.

Reply:
ah let me remove, good catch